Common/Tasks/multiplicityQa.cxx

```diff
@@ -29,55 +34,93 @@ using namespace o2;
+  Configurable<float> MaxMultFV0{"MaxMultFV0", 20000, "Max FV0 signal"};
+  Configurable<float> MaxMultFT0{"MaxMultFT0", 10000, "Max FT0 signal"};
+  Configurable<float> MaxMultFDD{"MaxMultFDD", 10000, "Max FDD signal"};
+  Configurable<float> MaxMultNTracks{"MaxMultNTracks", 1000, "Max Ntracks"};
+  Configurable<int> NBinsVertexZ{"NBinsVertexZ", 400, "max vertex Z (cm)"};
 
-  //For vertex-Z corrections
-  OutputObj<TProfile> hVtxProfV0M{TProfile("hVtxProfV0M", "", 150, -15, 15)};
-  OutputObj<TProfile> hVtxProfT0M{TProfile("hVtxProfT0M", "", 150, -15, 15)};
-  OutputObj<TProfile> hVtxProfZNA{TProfile("hVtxProfZNA", "", 150, -15, 15)};
-  OutputObj<TProfile> hVtxProfZNC{TProfile("hVtxProfZNC", "", 150, -15, 15)};
+  void init(InitContext&)
+  {
+    const AxisSpec axisEvent{10, 0, 10, "Event counter"};
+    const AxisSpec axisMultFV0{(int)NBinsMultFV0, 0, MaxMultFV0, "FV0 total amplitude"};
+    const AxisSpec axisMultFT0{(int)NBinsMultFT0, 0, MaxMultFT0, "FT0 total amplitude"};
+    const AxisSpec axisMultFDD{(int)NBinsMultFDD, 0, MaxMultFDD, "FDD total amplitude"};
+    const AxisSpec axisMultNTracks{(int)NBinsMultNTracks, 0, MaxMultNTracks, "Track counter"};
+    const AxisSpec axisVertexZ{(int)NBinsVertexZ, -20, 20, "Vertex Z (cm)"};
 
-  OutputObj<TProfile> hMultNtrackletsVsV0M{TProfile("hMultNtrackletsVsV0M", "", 50000, 0., 50000.)};
+    //Base histograms
+    histos.add("multiplicityQa/hEventCounter", "Event counter", kTH1D, {axisEvent});
+    histos.add("multiplicityQa/hRawFV0", "Raw FV0", kTH1D, {axisMultFV0});
+    histos.add("multiplicityQa/hRawFT0", "Raw FT0", kTH1D, {axisMultFT0});
+    histos.add("multiplicityQa/hRawFDD", "Raw FDD", kTH1D, {axisMultFDD});
+    histos.add("multiplicityQa/hRawNTracks", "Raw NTracks", kTH1D, {axisMultNTracks});
+    histos.add("multiplicityQa/hZeqFV0", "vtx-z eq FV0", kTH1D, {axisMultFV0});
+    histos.add("multiplicityQa/hZeqFT0", "vtx-z eq FT0", kTH1D, {axisMultFT0});
+    histos.add("multiplicityQa/hZeqFDD", "vtx-z eq FDD", kTH1D, {axisMultFDD});
+    histos.add("multiplicityQa/hZeqNTracksPV", "vtx-z eq NTracks", kTH1D, {axisMultNTracks});
 
-  Configurable<bool> isMC{"isMC", 0, "0 - data, 1 - MC"};
-  Configurable<int> selection{"sel", 7, "trigger: 7 - sel7, 8 - sel8"};
+    //Vertex-Z profiles for vertex-Z dependency estimate
+    histos.add("multiplicityQa/hVtxZFV0A", "Av FV0A vs vertex Z", kTProfile, {axisVertexZ});
+    histos.add("multiplicityQa/hVtxZFT0A", "Av FT0A vs vertex Z", kTProfile, {axisVertexZ});
+    histos.add("multiplicityQa/hVtxZFT0C", "Av FT0C vs vertex Z", kTProfile, {axisVertexZ});
+    histos.add("multiplicityQa/hVtxZFDDA", "Av FDDA vs vertex Z", kTProfile, {axisVertexZ});
+    histos.add("multiplicityQa/hVtxZFDDC", "Av FDDC vs vertex Z", kTProfile, {axisVertexZ});
+    histos.add("multiplicityQa/hVtxZNTracksPV", "Av NTracks vs vertex Z", kTProfile, {axisVertexZ});
+  }
 
   void process(soa::Join<aod::Collisions, aod::EvSels, aod::Mults>::iterator const& col)
```

Comment:
What about name it `processRun3` and incorporate the proper `SWITCH` because it is only intended for Run3

Reply:
This I can do! Let me implement that in the next commit.